packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-drilldown.vue

```diff
@@ -82,7 +82,7 @@
 				<Button label="Save as" outlined severity="secondary" @click="showSaveModal = true" />
 				<Button :disabled="isSaveDisabled" label="Save" @click="onSaveConfiguration" />
 			</template>
-			<Accordion multiple :active-index="[0, 1, 2]">
```

Comment:
I see many many examples around the app of us using a single `Accordion` for multiple `AccordionTab`s
Im not sure why we need to break this down into three Accordions?

Reply:
because it's been hacked together over the years. and the model needs a better refactoring that what we got.